web-registry/src/lib/rdf.ts

```diff
@@ -66,3 +89,9 @@ export const qudtUnitMap = {
   'unit:HA': 'hectares',
   'unit:AC': 'acres',
 };
+
+export function getProjectShapeIri(creditClassId?: string | null): string {
+  return creditClassId
+    ? `http://regen.network/${creditClassId}-ProjectShape`
+    : 'http://regen.network/ProjectPageShape';
+}
```

Comment:
@haveanicedavid you can re-use this function to query the appropriate SHACL graph for validating project data
basically if we have some on chain `creditClassId`, we will fetch the corresponding SHACL graph, eg `http://regen.network/C01-ProjectShape`